studiopress/simple-social-icons
# Add TripAdvisor and WhatsApp icons

**Summary of change:**
Add TripAdvisor and WhatsApp icons

**Have the changes in this PR been added to the documentation for this project?**
Yes

**How to test:**

1. Look for TripAdvisor and WhatsApp URI field in widget
2. Add a URI to the TripAdvisor and WhatsApp URI field
3. Check that the TripAdvisor and WhatsApp icons had appeared
